package.json

```diff
@@ -11,6 +11,7 @@
     "clean:workspaces": "turbo clean",
     "db:generate": "turbo db:generate",
     "db:push": "turbo db:push db:generate",
+    "db:studio": "turbo db:studio",
```

Comment:
no reason to run this with turbo right? no caching, no task dep.

I'd just run it with a `pnpm --filter db db:studio` or something

Reply:
That's a great point actually